lib/routes/content.js

```diff
@@ -85,38 +84,39 @@ const routes = application => {
      * @param {object} next - Express callback
     */
     app.get(`/${config.content.base}/:type/${config.content.actions.add}`, (req, res, next) => {
-      const type = utils.singleItem('id', req.params.type.toLowerCase(), types);
-      const workflow = workflows.workflow(type, allflows, config, req);
-      const errors = _.get(req.session, `form.content.add[${req.params.type.toLowerCase()}].errors`, {});
-      const values = _.get(req.session, `form.content.add[${req.params.type.toLowerCase()}].content`, {});
+      utils.fill(types, req.params.type.toLowerCase(), references, database).then(type => {
```

Comment:
We don't have access to database in content-types. Secondly, the content in content-types can change while the server is running and merge will not be able to reflect those changes as it is called on initialization.